### PR TITLE
chore: save only camera timestamps

### PR DESCRIPTION
**Pull request by versey-sherry**

### CODE REVIEW

File: azure_acquire/util.py [hunk 1]
```diff
@@ -239,12 +239,14 @@ def capture_from_azure(k4a, filename_prefix, recording_length, save_ir=True,
     finally:
         k4a.stop()
         system_timestamps = np.array(system_timestamps)
```
Comment (wingillis) [inline azure_acquire/util.py]: delete this line

File: azure_acquire/util.py [hunk 2]
```diff
@@ -239,12 +239,14 @@ def capture_from_azure(k4a, filename_prefix, recording_length, save_ir=True,
...
+        # change from microsecond to millisecond
+        device_timestamps = np.array(device_timestamps)/1000
         
         # np.save(filename_prefix+'.system_timestamps.npy',system_timestamps)
         # np.save(filename_prefix+'.device_timestamps.npy',device_timestamps)
         
-        np.savetxt(os.path.join(filename_prefix, 'depth_ts.txt'), system_timestamps, fmt = '%f')
-        np.savetxt(os.path.join(filename_prefix, 'device_ts.txt'),device_timestamps, fmt = '%f')
+        # np.savetxt(os.path.join(filename_prefix, 'system_ts.txt'), system_timestamps, fmt = '%f')
```
Comment (wingillis) [inline azure_acquire/util.py]: you can delete these lines

File: setup.py [hunk 3]
```diff
@@ -24,6 +24,7 @@ def get_version(rel_path):
     version=get_version('azure_acquire/__init__.py'),
     packages=find_packages(),
     platforms=['mac', 'unix'],
+    install_requires=['click', 'opencv-python', 'numpy', 'matplotlib', 'jupyter', 'pyk4a', 'jupyter'],
```
Comment (wingillis) [inline setup.py:27]: make sure the comments I made in the previous PR get merged into this PR too

**Review by wingillis — CHANGES_REQUESTED** [review]

File: azure_acquire/util.py (hunk 2 above)
Comment (versey-sherry) [reply azure_acquire/util.py]: e927585faa7278b18e2abf2f07dcd4e2bbc7710f

File: azure_acquire/util.py (hunk 1 above)
Comment (versey-sherry) [reply azure_acquire/util.py]: e927585faa7278b18e2abf2f07dcd4e2bbc7710f

**Pull request merged**